src/styles/blocks/review.scss

```diff
@@ -0,0 +1,30 @@
+.review {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+
+    text-align: right;
+    line-height: 16px;
+
+    margin-block: 0;
+    padding-bottom: 24px;
+  }
+
+  &__price {
+    font-size: 12px;
```

Comment:
Check if the `font-size` of 12px is consistent with the design specifications. It's important that all font sizes match the design for visual consistency.